Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN composer install \
 # Copy Project
 COPY modules/custom modules/custom
 COPY sites/ sites/
+COPY ./flysystem/FieldMigration.php /var/www/html/modules/contrib/flysystem/src/Form/FieldMigration.php
```

Comment:
<I don't have an emoji for "I'm being really nitpicky"> I think we can remove the preceding `./` for consistency with the lines above

Reply:
Removed this line anyways :)